### DISCUSSION TITLE
Use of Docker/Validator

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#469] [Q&A] [answered]
Hi @shaselton-usds 
Docker has updated their site and now charges a fee (see below) for employers with more than 250 employees.  During the webinar it was stated that docker was free.  

If this isn't correct could you point me to the right application to download please. Thanks.

**Comment by olderthandirt5** [comment, marked as answer]
Since no one else has responded, I will confirm your understanding is correct AFAIK. This change was fairly recent, 8/31/2021 with a grace period until 1/31/2022. A single developer can purchase a Personal plan for $5/month billed at $60 annually.
Source: https://www.docker.com/pricing/

I am not affiliated with Docker but have used it at several companies.

**Reply by Ginger228** [reply]
Thanks for your help.

**Reply by shaselton-usds** [reply]
Thanks @olderthandirt5. For personal use, it is free. There are tiered prices that are typical for PaaS products if Docker were to be used as part of a company's development suite.
